src/components/GetActivities.js

```diff
@@ -0,0 +1,44 @@
+import eventList from "../data/events.json"
+import Title from "./Title"
+import { FaTwitter, FaDiscord } from "react-icons/fa"
+
+const events = eventList.events
+const styles = "transition-all ease-in cursor-pointer"
```

Comment:
Are you sure tailwind will pick up these styles? Also can you please not use transition-all, but transition only for the properties that will change? Please also add the will-change, property for those elements